src/cli/argv.ts

```diff
@@ -99,5 +99,19 @@ export async function processArgs(args, env) {
             },
             []
         )
+        .command('provider_accounts', 'List provider accounts', (yargs) => {
+                return yargs
+                    .option('providerId', {type: 'string', demand: false})
+                    .option('status', {type: 'string', demand: false})
+            }, async (argv) => {
+                try {
+                    let result = await tasks.providerAccounts(argv.providerId, argv.status)
```

Comment:
Definitely the right way to go about this for now. In future, I think running commands in this big yargs wrapper is going to get messy. I'd rather work out a way to return everything that needs to be run then do a `Promises.all` or something like that.